code/datums/jobs/access.dm

```diff
@@ -263,27 +263,27 @@ proc/get_all_job_icons()
 
 /proc/get_marine_jobs()
 		return list(
-				"Captain",
```

Comment:
same here just make a GLOB so you arent instancing a new list everytime this proc is called

Reply:
Done.

Reply:
Done.